### PR TITLE
Add pre-push to only run tests before pushing

### PR DESCRIPTION
**Pull request by sameetandpotatoes**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,6 +16,8 @@ Note that the scripts must be executable for the hooks to run.
 ```
 chmod +x scripts/pre-commit
 ln -s -f ../../scripts/pre-commit .git/hooks/pre-commit
+chmod +x scripts/post-commit
```
Comment (wavyllama) [inline README.md]: Is this supposed to be pre-push?

Comment (sameetandpotatoes) [reply README.md]: Good catch

**Comment by sameetandpotatoes** [comment]
@wavyllama updated

**Comment by wavyllama** [comment]
LGTM :+1:

**Pull request merged**